content/academy/apify_platform/publishing_actors_on_apify_store/seo_and_promotion.md

```diff
@@ -88,16 +87,14 @@ The README․md file represents a complete description of your actor, intended f
 You have lots of room for SEO optimization in the README, without compromising readability for your users. Make sure the text includes other variants of what potential users of your actor might search for. A good place for these is in the captions and headlines but keywords in the body text will also help. It’s also a good idea to repeat the name of the actor several times.
 
 > **GOOD**: You can use this Amazon Crawler with your code (Python, PHP, Java) to crawl Amazon without the API...
-<br/>
-> **AVOID**: Follow the instructions in the Readme and if you have any comments, see the FAQ section...
+> <br/> **AVOID**: Follow the instructions in the Readme and if you have any comments, see the FAQ section...
 
-An important part of SEO is internal and external linking. Don’t be shy -  add links to your README leading to similar [actors](https://apify.com/store) or any interesting article in [Apify Help](https://help.apify.com/) that could assist your users. If your actor is scraping a certain website, make sure to add a link to that website. If the actor uses any external libraries, make sure to link their pages.
+An important part of SEO is internal and external linking. Don’t be shy - add links to your README leading to similar [actors](https://apify.com/store) or any interesting article in [Apify Help](https://help.apify.com/) that could assist your users. If your actor is scraping a certain website, make sure to add a link to that website. If the actor uses any external libraries, make sure to link their pages.
 
 If you’ve created other actors, it’s completely fine to pop in a promo for them at the end. It’s even better if you edit your old actors and put in a link to your new actor. We’ll take care of extra promotion on the [Apify homepage](https://apify.com/) or [Apify Store](https://apify.com/store) - especially if you’ve done a great job of describing your actors!
 
 > **GOOD**: Like my <a href="#">Amazon Crawler</a>? See my <a href="#">Amazon Review Crawler</a>.
-<br/>
-> **AVOID**: Check out my Amazon Crawler.
+> <br/> **AVOID**: Check out my Amazon Crawler.
```

Comment:
What's the `<br />` HTML tag doing here? It is generally bad practice to use HTML in Markdown unless there is a good reason for doing so.

Reply:
The `<br/>` is ensuring the "AVOID" text stays on a new line without breaking the "card" style. If I simply used space, then this "card-look" would split in two. See the image attached for reference.

<img width="1279" alt="Screenshot 2022-10-14 at 11 33 15 AM" src="https://user-images.githubusercontent.com/70678259/195815123-5f089a84-ae82-4ff7-8174-429ab3ba7d7f.png">

Reply:
Just add a return between the two quotes to make them not merge together instead of using `<br>`

Reply:
Ok makes sense